### ISSUE TITLE
Please update the wiki to include info about "real-time flavour" ISO

### ISSUE THREAD

**Issue by argumemnon** [vmware/photon#1383]
### Is your feature request related to a problem? Please describe.

No info on "real-time flavour" iso or what it is named such. 

### Describe the solution you'd like

Can you please update the wiki page about downloads to include information about the "real-time flavour" ISO and how it differs from the other x86_64 ISO downloads?

**Comment by dcasota** [comment]
@argumemnon Beside the wiki, there are some docs entries good to know about Photon OS real-time flavor.
https://vmware.github.io/photon/docs/overview/flavours/
https://vmware.github.io/photon/docs/administration-guide/photon-real-time-operating-system/

https://wiki.linuxfoundation.org/realtime/start
https://wiki.linuxfoundation.org/realtime/preempt_rt_versions
( thanks to @srivatsabhat )

In a VMware virtualization environment, the [Open VM Tools](https://docs.vmware.com/en/VMware-Tools/12.1.0/com.vmware.vsphere.vmwaretools.doc/GUID-8B6EA5B7-453B-48AA-92E5-DB7F061341D1.html) should be considered as well.
Unfortunately it is not easy to distinguish between RT and non-RT environment, see [lamw's entry](https://github.com/vmware/photon/issues/1201).

Which additional information would be helpful for your project using Photon OS real-time flavor ?

**Comment by sshedi** [comment]
> Unfortunately it is not easy to distinguish between RT and non-RT environment, see https://github.com/vmware/photon/issues/1201.

Not really. Here is my `uname -a` output from latest linux-rt kernel installation.
```
root@ph3dev:~ # uname -a
Linux ph3dev 4.19.268-1.ph3-rt #1-photon SMP PREEMPT RT Sat Dec 17 14:09:13 UTC 2022 x86_64 GNU/Linux
```
